ansible/roles/projects/ovn-kubernetes/test/tasks/deploy-test-pods.yml

```diff
@@ -37,3 +23,28 @@
           - "pod_ip in pod_ovn_annotation.default.ip_address"
     - set_fact:
         pod_ips: "{{ pod_ips + [pod_ip] }}"
+    - name: Verfiy VF in pod
+      collections:
+        - community.kubernetes
+      block:
+        - name: Read pod VF device vendor
+          k8s_exec:
+            namespace: default
+            pod: "{{ pod_name }}"
+            kubeconfig: "{{ kubeconfig }}"
+            context: "{{ k8s_context }}"
+            command: cat /sys/class/net/eth0/device/vendor
+          register: vf_vendor
+        - name: Read pod VF device ID
+          k8s_exec:
+            namespace: default
+            pod: "{{ pod_name }}"
+            kubeconfig: "{{ kubeconfig }}"
+            context: "{{ k8s_context }}"
+            command: cat /sys/class/net/eth0/device/device
+          register: vf_id
```

Comment:
To not limit the CI to connectX5 cards only, we can only check for vendor id

Reply:
fixed